src/Context/GetoptParser.php

```diff
@@ -265,12 +270,14 @@ protected function splitLongOptionInput($input)
      * @return bool
      *
      */
-    protected function longOptionRequiresValue($option, $value, $name)
+    protected function longOptionRequiresValue($option, $value, $name=null)
```

Comment:
I think the Aura project's style guidelines would prefer spaces around the equals sign:
`, $name = null`